codeHF/workflows.yml

```diff
@@ -156,6 +172,11 @@ workflows:
   o2-analysis-hf-task-lc:
     dependencies: o2-analysis-hf-lc-candidate-selector
 
+  o2-analysis-hf-task-lb:
+    options:
+      mc: "--doMC"
```

Comment:
This should be removed together with replacing the `--doMC` option by a `processMC` switch ASAP.